docs/guides/latest/workloads/daemonset.md

```diff
@@ -0,0 +1,759 @@
+    Mounts:
+      /etc/stash/repository/secret from stash-secret-volume (rw)
+      /source/data from source-data (rw)
+      /tmp from tmp-dir (rw)
+      /var/run/secrets/kubernetes.io/serviceaccount from default-token-4tzgg (ro)
+Containers:
+  busybox:
+    Container ID:  docker://792f32e4fdf758084489c4f0a340f0dbde3b84189ca5550ee3ad93d1fbb97cef
+    Image:         busybox
+    Image ID:      docker-pullable://busybox@sha256:c94cf1b87ccb80f2e6414ef913c748b105060debda482058d2b8d0fce39f11b9
+    Port:          <none>
+    Host Port:     <none>
+    Args:
+      sleep
+      3600
+    State:          Running
+      Started:      Wed, 26 Jun 2019 14:25:56 +0600
+    Ready:          True
+    Restart Count:  0
+    Environment:    <none>
+    Mounts:
+      /source/data from source-data (rw)
+      /var/run/secrets/kubernetes.io/serviceaccount from default-token-4tzgg (ro)
+Conditions:
+  Type              Status
+  Initialized       True 
+  Ready             True 
+  ContainersReady   True 
+  PodScheduled      True 
+Volumes:
+  source-data:
+    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
+    ClaimName:  demo-pvc
+    ReadOnly:   false
+  tmp-dir:
+    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
+    Medium:     
+    SizeLimit:  <unset>
+  stash-podinfo:
+    Type:  DownwardAPI (a volume populated by information about the pod)
+    Items:
+      metadata.labels -> labels
+  stash-secret-volume:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  gcs-secret
+    Optional:    false
+  default-token-4tzgg:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  default-token-4tzgg
+    Optional:    false
+QoS Class:       BestEffort
+Node-Selectors:  <none>
+Tolerations:     node.kubernetes.io/disk-pressure:NoSchedule
+                 node.kubernetes.io/memory-pressure:NoSchedule
+                 node.kubernetes.io/not-ready:NoExecute
+                 node.kubernetes.io/pid-pressure:NoSchedule
+                 node.kubernetes.io/unreachable:NoExecute
+                 node.kubernetes.io/unschedulable:NoSchedule
+Events:
+  Type    Reason     Age   From               Message
+  ----    ------     ----  ----               -------
+  Normal  Scheduled  82s   default-scheduler  Successfully assigned demo/stash-recovered-dqlrb to minikube
+  Normal  Pulled     80s   kubelet, minikube  Container image "suaas21/stash:vs_linux_amd64" already present on machine
+  Normal  Created    80s   kubelet, minikube  Created container stash-init
+  Normal  Started    80s   kubelet, minikube  Started container stash-init
+  Normal  Pulled     69s   kubelet, minikube  Container image "busybox" already present on machine
+  Normal  Created    69s   kubelet, minikube  Created container busybox
+  Normal  Started    69s   kubelet, minikube  Started container busybox
+```
+
+You will see that `stash-init` container has been injected.
```

Comment:
Notice the `Init-Containers` section. We can see that init-container `stash-init` has been injected which is running `restore` command.